Doc/library/codecs.rst

```diff
@@ -473,7 +516,7 @@ The base :class:`Codec` class defines these methods which also define the
 function interfaces of the stateless encoder and decoder:
 
 
-.. method:: Codec.encode(input[, errors])
+.. method:: Codec.encode(input[, errors='strict'])
```

Comment:
```suggestion
.. method:: Codec.encode(input, errors='strict')
```

If we give the default we don't need the square brackets to say that the param is optional.